ci/cpu/build.sh

```diff
@@ -34,13 +41,14 @@ conda list --show-channel-urls
 function build_pkg {
   # Build pkg
   gpuci_logger "Start conda build for '${1}'..."
-  conda build -c ${CONDA_USERNAME:-rapidsai-nightly} -c nvidia -c conda-forge --python=${PYTHON_VERSION} ${1}
+  conda build -c ${CONDA_USERNAME:-rapidsai-nightly} -c nvidia -c conda-forge \
+              --python=${PYTHON_VER} --append-file ${CONDA_CONFIG_FILE} ${1}
```

Comment:
I'm not sure `--append-file` works here, I tried running this manually and get:
```
$ conda build -c rapidsai-nightly -c nvidia -c conda-forge --python=3.6 --append-file conda/recipes/nightly-versions.yaml conda/recipes/rapids-xgboost/
Error: Failed to render jinja template in /workspace/integration/conda/recipes/rapids-xgboost/meta.yaml:
'nccl_version' is undefined
```
Using `-m` seems to work though

Reply:
Yeah it should be `-m` I accepted the wrong change when pulling in my stash